src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -204,6 +204,8 @@ public enum SyntaxKind : ushort
         DataKeyword = 8441,
         WithKeyword = 8442,
         InitKeyword = 8443,
+        RecordKeyword = 8444,
+
         /// when adding a contextual keyword following functions must be adapted:
```

Comment:
>when adding a contextual keyword following functions must be adapted: [](start = 12, length = 69)

Please make sure this is taken care of

Reply:
Checked, everything seems to be in order.